app/screens/more_channels/more_channels.js

```diff
@@ -144,18 +147,18 @@ class MoreChannels extends PureComponent {
 
     filterChannels = (channels, term) => {
         return channels.filter((c) => {
-            return (c.name.toLowerCase().indexOf(term) !== -1 || c.display_name.toLowerCase().indexOf(term) !== -1);
+            return (c.name.toLowerCase().includes(term) || c.display_name.toLowerCase().includes(term));
         });
     };
 
     searchProfiles = (text) => {
         const term = text.toLowerCase();
 
         if (term) {
-            const channels = this.filterChannels(this.state.channels, term);
+            const channels = this.filterChannels(this.props.channels, term);
             this.setState({
                 channels,
-                term: text,
+                term,
```

Comment:
At first, I had a doubt that the text on search box will always change to lowercase, even if the inputted text is with uppercase. 
But it works fine after I tested a bit on iOS simulator.
<img width="329" alt="screen shot 2018-03-07 at 9 49 49 pm" src="https://user-images.githubusercontent.com/5334504/37095684-902ab122-2251-11e8-847b-2641a81dec35.png">

Reply:
This is for the More Channels screen, not the search box. I think the searching for the search box is always case insensitive since it's done on the server.

Reply:
lol, I was out of my mind, I should have tested that on More Channels screen :)

I mean, it's weird that the user is typing in uppercase but then the input box is always showing the letter in lower case. Not a big deal, and might not be an issue.

![screen shot 2018-03-07 at 9 49 49 pm](https://user-images.githubusercontent.com/5334504/37098790-b7db31da-2259-11e8-8648-37ec78652060.gif)